src/main/scopes/js/metrics/token-metric.ts

```diff
@@ -63,9 +67,40 @@ export class TokenMetric extends ScopeMetric {
       this.instrumentedPackage.version
     )
 
+    let tokenName = this.jsToken.name
+    const tokenAccessPath = this.jsToken.accessPath
+
+    // Handle renamed tokens
+    if (this.matchingImport.rename !== undefined) {
+      tokenName = tokenName.replace(this.matchingImport.rename, this.matchingImport.name)
+      // replace import name in access path
+      tokenAccessPath[0] = this.matchingImport.name
+    }
+
+    const subs = new Substitution()
+    // redact complex values
+    tokenAccessPath.forEach((segment) => {
+      if (segment instanceof ComplexValue) {
+        tokenName = safeStringify(tokenName).replace(
```

Comment:
minor: Same comment here about being able to move the safeStringify call up to the assignment statement.